docs/running-on-kubernetes.md

```diff
@@ -1811,6 +1811,50 @@ spec:
   queue: default
 ```
 
+#### Using Apache YuniKorn as Customized Scheduler for Spark on Kubernetes
+
+[Apache YuniKorn](https://yunikorn.apache.org/) is a resource scheduler for Kubernetes that provides advanced batch scheduling
+capabilities, such as job queuing, resource fairness, min/max queue capacity and flexible job ordering policies.
+For available Apache YuniKorn features, please refer to [this doc](https://yunikorn.apache.org/docs/next/get_started/core_features).
```

Comment:
We cannot use the version number alias `next` here because it will be fragile in the future.
- We need to use a concrete version `v1.0.0` link instead.
- However, Apache YuniKorn doesn't provide `1.0.0` yet.
  - https://yunikorn.apache.org/docs/1.0.0/ is broken.
  - The only latest version seems to be https://yunikorn.apache.org/docs/0.12.2/ .

We need Apache YuniKorn community's help here, @yangwwei .

Reply:
hi @dongjoon-hyun This is how the doc site works, next -> is the current under-development version, we shouldn't use this, that's a good point; but I think we can use the latest stable version: this points to https://yunikorn.apache.org/docs/. Only the past versions are accessible via https://yunikorn.apache.org/docs/{VERSION_NUM}, that's why you did not see 1.0.0 there, 1.0.0 is the current stable version.

If we use a hard-coded version, e.g 1.0.0 here, we will need to come back to update the doc quite often, I don't feel that is good. So my question is: is it better to use the latest stable version here or a hard-coded version that will need updates over time? Please let me know, thanks!